### PR TITLE
chore: Changelog & pot file updated with dynamic-css file name

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Changelog updated
- Pot file updated
- Related Posts - Dynamic CSS file name renamed

### Types of changes
- Non breaking change

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,7 +1,10 @@
 v3.4.0 (Unreleased)
+- Improvement: “Related Posts” new section in single posts layout.
```
Comment (premanshup) [inline changelog.txt:2]:
```suggestion
- New: “Related Posts” feature for single post layout.
```

File: changelog.txt [hunk 2]
```diff
@@ -23,6 +26,7 @@ v3.4.0 (Unreleased)
 - Fix: Accessibility - Tab navigation does not highlight menu with submenus.
 - Fix: JavaScript error when clicking on the Dropdown Toggle for Sticky Header on Mobile.
 - Fix: Scroll does not work in Off-Canvas.
+- Fix: Builder – Spacing jerk issue in customizer while adjusting Stack on & Icon Spacing options in Social element.
```
Comment (premanshup) [inline changelog.txt:29]:
```suggestion
- Fix: Builder – Spacing jerk issue in customizer while adjusting Stack on mobile & Icon Spacing options in Social element.
```
@imnavanath Should this be `stack on mobile`?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 1 above)
Comment (imnavanath) [reply changelog.txt]: Done

**Review by premanshup — APPROVED** [review]

File: changelog.txt (hunk 2 above)
Comment (imnavanath) [reply changelog.txt]: Done

**Pull request merged**